tests/application/cms/test_views.py

```diff
@@ -799,3 +800,30 @@ def test_copy_measure_page(test_app_client, logged_in_dev_user):
 
     assert page.find("h1").text == "Edit page"
     assert page.find("input", attrs={"name": "title"})["value"] == "COPY OF Test Measure Page"
+
+
+def test_list_measure_versions_shows_expected_links(test_app_client, logged_in_dev_user):
```

Comment:
Maybe name this `test_measure_version_history_page` given that it's the only test for that view?

Reply:
👍